content/applications/sales/point_of_sale/payment_methods/terminals/razorpay.rst

```diff
@@ -0,0 +1,75 @@
+========
+Razorpay
+========
+
+Connecting a Razorpay payment terminal allows you to offer a fluid payment flow to your customers
+and ease the work of your cashiers.
+
+.. seealso::
+   - :doc:`Use Razorpay as payment provider. <../../../../finance/payment_providers/razorpay>`
+
+Configuration
+=============
+
+.. _razorpay/credentials:
+
+Locate your Razorpay credentials
+--------------------------------
+
+`Create a Razorpay account <https://razorpay.com/docs/payments/easy-create-account/>`_ and set it up
+on their website.
+
+You need the following credentials to set up the payment method in Odoo:
+
+- `API key <https://razorpay.com/docs/payments/dashboard/account-settings/api-keys/>`_
+- Razorpay username
+- Razorpay device serial number
+
+   - You can find your payment terminal serial number under the device or on `Razorpay's dashboard
+     <https://dashboard.razorpay.com/>`_.
+
+Configure the payment method
+----------------------------
+
+#. :doc:`Activate the POS Razorpay module <../../../../general/apps_modules>` to enable the
+   payment terminal.
+#. :doc:`Create the related payment method <../../payment_methods>` by going to
+   :menuselection:`Point of Sale --> Configuration --> Payment Methods`.
+
+   #. Set the :guilabel:`Journal` type as :guilabel:`Bank`.
+   #. Select :guilabel:`Razorpay` in the :guilabel:`Use a Payment Terminal` field.
+   #. Enter your username in the :guilabel:`Razorpay Username` field and your device's serial number
+      in the :guilabel:`Razorpay Device Serial No` field.
+   #. Fill in the :guilabel:`Razorpay API Key` field with the :ref:`Razorpay API key
+      <razorpay/credentials>`.
+   #. Set the :guilabel:`Razorpay Allowed Payment Modes` according to your needs.
+
+   .. image:: razorpay/create-method-razorpay.png
+      :alt: Razorpay connection form
+
+   .. note::
+      - Click :guilabel:`Identify Customer` to allow this payment method **exclusively** for
+        identified customers. For any unidentified customers to be able to pay with Razorpay, leave
+        the :guilabel:`Identify Customer` field unchecked.
```

Comment:
I think this should be described on the Payment methods doc page instead since it's not specific to Razorpay